pkg/influx/api.go

```diff
@@ -27,8 +27,15 @@ type API struct {
 	recorder Recorder
 }
 
-func (a *API) Register(server *server.Server, authMiddleware middleware.Interface) {
-	server.HTTP.Handle("/api/v1/push/influx/write", authMiddleware.Wrap(http.HandlerFunc(a.handleSeriesPush)))
+func (a *API) Register(server *server.Server, enableAuth bool) {
```

Comment:
Could you add tests to check enabling and disabling auth work as expected?

Reply:
Since this is a small change that will unblock other work, we think it's ok to merge this now and add the test in a later PR (since it will require some refactoring to create the foundation necessary to do so)